File: Javascript/Uri1038_Lanche.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split(' ');

var cod = parseInt(lines.shift());
var qtd = parseInt(lines.shift());

if (cod == 1){
    var tot = qtd * 4.00;
}

else if(cod == 2){
    var tot = qtd * 4.50;
}

else if(cod == 3){
    var tot = qtd * 5.00;
}

else if(cod == 4){
    var tot = qtd * 2.00;
}

else if(cod == 5){
    var tot = qtd * 1.50;
}

console.log("Total: R$ " + tot.toFixed(2));

File: Python/Uri1038_Lanche.py
leitura = input().split()

codigo = int(leitura[0])
quantidade = int(leitura[1])

if codigo == 1:
    result = 4.00 * quantidade
    print('Total: R$ {:.2f}'.format(result))

elif codigo == 2:
    result = 4.50 * quantidade
    print('Total: R$ {:.2f}'.format(result))

elif codigo == 3:
    result = 5.00 * quantidade
    print('Total: R$ {:.2f}'.format(result))

elif codigo == 4:
    result = 2.00 * quantidade
    print('Total: R$ {:.2f}'.format(result))

elif codigo == 5:
    result = 1.50 * quantidade
    print('Total: R$ {:.2f}'.format(result))